tycho-its/projects/target.maven.autofeature/test.target

```diff
@@ -17,10 +17,6 @@
 			</dependencies>
 		</location>
 		<location includeSource="true" missingManifest="generate" type="Maven">
-
```

Comment:
This change seems to indicate that not the latest m2e-snapshot is used here and what makes the test fail